src/main/java/hudson/maven/MavenModuleSet.java

```diff
@@ -1365,6 +1365,39 @@ public String getDisplayName() {
             return Messages.MavenModuleSet_DiplayName();
         }
 
+        /**
+         * Needed if it wants Maven projects are categorized in Jenkin 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string it represents a ItemCategory identifier.
+         */
+        public String getCategoryId() {
+            return "itemcategory-standaloneprojects";
+        }
+
+        /**
+         * Needed if it wants Maven projects are categorized in Jenkin 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string with the Item description.
+         */
+        public String getDescription() {
```

Comment:
@oleg-nenashev It seems this plugins is not using this kind of annotations. Could @aheritier confirm it?